### PR TITLE
Remove vercel and setup ReadTheDocs

### PR DESCRIPTION
**Pull request by Esteban82**
Applying the changes discussed in #13.

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "mambaforge-4.10"
+  jobs:
+    pre_build: # Generate api stub files before building
+      - make -C doc api
```
Comment (seisman) [inline .readthedocs.yaml:14]:
```suggestion
```

File: .readthedocs.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# formats:
+#    - pdf
+
+# Optionally declare the Python requirements required to build your docs
+conda:
+  environment: environment.yml
+python:
+   install:
+   - method: pip
+     path: .
```
Comment (seisman) [inline .readthedocs.yaml:30]:
```suggestion
```

File: .readthedocs.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "mambaforge-4.10"
+
+# Build documentation in the doc/ directory with Sphinx
+sphinx:
+   configuration: doc/conf.py
```
Comment (seisman) [inline .readthedocs.yaml:15]:
```suggestion
# Build documentation in the docs/ directory with Sphinx
sphinx:
   configuration: docs/conf.py
```

**Comment by Esteban82** [comment]
I think that now it is working fine.

**Review by seisman — APPROVED** [review]

**Pull request merged**